### ISSUE TITLE
[ICON REQUEST]: <icon name>

### ISSUE THREAD

**Issue by Dum6o** [devicons/devicon#1398]
### I have searched through the issues and didn't find my problem.

- [X] Confirm

### About the icon

Jira icon (new version)

### Links and sources

Jira: https://www.atlassian.com/software/jira
FontAwesome: https://fontawesome.com/icons/jira?f=brands
```html
<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 496 512">
<!--! Font Awesome Pro 6.2.0 by @fontawesome - https://fontawesome.com License - https://fontawesome.com/license (Commercial License) Copyright 2022 Fonticons, Inc. -->
<path d="M490 241.7C417.1 169 320.6 71.8 248.5 0 83 164.9 6 241.7 6 241.7c-7.9 7.9-7.9 20.7 0 28.7C138.8 402.7 67.8 331.9 248.5 512c379.4-378 15.7-16.7 241.5-241.7 8-7.9 8-20.7 0-28.6zm-241.5 90l-76-75.7 76-75.7 76 75.7-76 75.7z"/>
</svg>
```

### Additional information

Atlassian Jira logo had change long time ago. Here I can still find only old one. Would be great to update it 🙂

**Comment by Panquesito7** [comment]
Hey there! Would you like to update the icon yourself? 🙂

**Comment by GuilhermeFernandes01** [comment]
Hi @Dum6o, according to [here](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#not-allowed-not-exhaustive), Jira icon is not allowed in this project.

**Comment by Snailedlt** [comment]
> Hi @Dum6o, according to [here](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#not-allowed-not-exhaustive) Jira icon is not allowed in this project.

Nice catch!
However since it's updating an icon, and not requesting a new one, It's allowed. All icons are kept for backwards compatability.
Added some more info about updating icons to the wikipage though :) You can see the diff [here](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F/_compare/1c8d8607aa477da068fbffbfdd1312753beb3cff...609393d9814b365b2b365a0518f87013aff8520e)

Also worth mentioning that Jira fits under [special cases](https://github.com/devicons/devicon/wiki/What-Icons-Do-We-Accept%3F#special-cases-see-this-discussion-for-more-details) too, since it's a very well known and widely used technology used mainly by developers.

**Comment by Snailedlt** [comment]
# Official Atlassian logos can be found here
https://atlassian.design/resources/logo-library

## Direct download for Jira Logos
https://assets.ctfassets.net/8j5aqoy0ts8s/1y9Zan62VaterUggPDIDAx/7537cd8851766dfce84d0785dc8200d7/Jira.zip

## Previews
![logo-gradient-blue-jira](https://user-images.githubusercontent.com/43886029/196718412-23ee80c6-cdef-48ee-97e3-5c2d7e8831b0.svg)
![mark-gradient-blue-jira](https://user-images.githubusercontent.com/43886029/196718433-4629f15f-23d1-4897-8640-02acb154dbe2.svg)

**Comment by Snailedlt** [comment]
Agreed that it may confuse people. But those are examples, and not rules, and Jira is a good example of a project management tool. I'll add a little text before explaining that they are examples though :)

**Comment by Snailedlt** [comment]
@GuilhermeFernandes01 
I added this:
![image](https://user-images.githubusercontent.com/43886029/196782985-7ca5bc38-8ea0-4257-a930-c59b737aaf9e.png)
